src/Uno.UI/Extensions/TimeSpanExtensions.cs

```diff
@@ -75,5 +75,26 @@ internal static TimeSpan RoundToMinuteInterval(this TimeSpan time, int interval)
 
 			return time;
 		}
+
+		/// <summary>
+		/// Normalize TimeSpan between 0 and 24h
+		/// </summary>
+		/// <returns>Normalized TimeSpan</returns>
+		internal static TimeSpan NormalizeToDay(this TimeSpan value)
+		{
+			var day = TimeSpan.FromDays(1);
+
+			while (value < TimeSpan.Zero)
+			{
+				value += day;
+			}
+
+			while (value > day)
```

Comment:
edge case: `TimeSpan.FromHours(24).NormalizeToDay() -> TimeSpan.FromHours(24)`
Should be `>= day`.

Reply:
Unless this is intended?

Reply:
I think it's intended, full day is fine.

Reply:
Can you add a comment to clarify that.

Reply:
Yes, sure.